.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "build"]
 	path = build
-	url = git@github.com:upbound/build.git
+	url = https://github.com/upbound/build
+[submodule "githooks"]
+	path = githooks
+	url = https://github.com/FrangipaneTeam/githooks
```

Comment:
I would prefer to not force contributors to use conventional commit.

Reply:
release please use conventional commit for generate release.